Neos.Neos/Classes/Controller/LoginController.php

```diff
@@ -16,22 +16,29 @@
 use Neos\Error\Messages\Message;
 use Neos\Flow\Http\Cookie;
 use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Mvc\Controller\MvcPropertyMappingConfigurationService;
+use Neos\Flow\Mvc\FlashMessage\FlashMessageService;
 use Neos\Flow\Mvc\View\JsonView;
 use Neos\Flow\Security\Authentication\Controller\AbstractAuthenticationController;
 use Neos\Flow\Security\Exception\AuthenticationRequiredException;
 use Neos\Flow\Session\SessionInterface;
 use Neos\Flow\Session\SessionManagerInterface;
-use Neos\FluidAdaptor\View\TemplateView;
 use Neos\Neos\Domain\Repository\DomainRepository;
 use Neos\Neos\Domain\Repository\SiteRepository;
 use Neos\Neos\Service\BackendRedirectionService;
+use Neos\Neos\Ui\View\BackendFusionView;
```

Comment:
Is that a healthy dependency chain? Imho it would be better to copy the fusionView to neos/neos or to use a plain neos/fusion view instead.

In the end the BackendFusion view only adjusts the fusionPathPattern to 'resource://@package/Private/Fusion/Backend' so this is not worth a circular dependency.

Reply:
Oh sorry I missed that it was in the UI package...
Can revert it to how I had it before